curve25519-dalek/src/montgomery.rs

```diff
@@ -84,6 +84,27 @@ impl ConstantTimeEq for MontgomeryPoint {
     }
 }
 
+#[allow(clippy::needless_range_loop)]
+impl ConditionallySelectable for MontgomeryPoint {
+    fn conditional_select(a: &Self, b: &Self, choice: Choice) -> Self {
+        let mut bytes = [0u8; 32];
+        for i in 0..32 {
+            bytes[i] = u8::conditional_select(&a.0[i], &b.0[i], choice);
+        }
+        Self(bytes)
```

Comment:
I'd imagine this would work instead:

```suggestion
        Self(<[u8; 32]>::conditional_select(&a.0, &b.0, choice)
```

Reply:
https://docs.rs/subtle/latest/subtle/trait.ConditionallySelectable.html Is not implemented for arrays, I can open a PR to subtle though :)

Reply:
The impl is gated under the `const-generics` feature of `subtle`. Unfortunately it seems the docs.rs build doesn't enable it.

Reply:
I'll at least open a PR for that lol :)

Reply:
And really now that `subtle` is MSRV 1.60, it should probably just be enabled by default.

Reply:
Fixed :)